docs/UserGuide.adoc

```diff
@@ -165,18 +177,27 @@ Profile Picture indicates the profile picture's file path
 
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 cp/Software Engineer cc/Google pp//home/john/Desktop/John.jpg` +
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 cp/Software Engineer cc/Google` +
 +
 On running the above command, you should see the following success message: +
 
  New person added: John Doe Phone: 98765432 Email: johnd@example.com Address: John street, block 123, #01-01 Current Position: Software Engineer Company: Google Skills:
 
+.Success message after adding John Doe
+
+image::addSuccess1.png[width="800", align="center"]
```

Comment:
Don't need this image because we have mentioned the same thing above. also the image is not clear. Just remove it.